packages/ffe-searchable-dropdown-react/less/ffe-multiselect-searchable-dropdown.less

```
@@ -0,0 +1,203 @@
+.ffe-searchable-dropdown-multiselect {
+    @check-icon-width: 24px;
```

css custom properties > less
```suggestion
    --check-icon-width: 24px;
```